kratos/tests/test_object_printing.py

```diff
@@ -9,9 +9,10 @@ def GetFilePath(fileName):
 # The expected definitions are here to make the handling of the
 # multiline-strings easier (no need to deal with indentation)
 prop_str = '''Properties
+Id : 5
     VISCOSITY : 5.3
     DENSITY : -95.3
-This properties contains 0 tables'''
```

Comment:
Maybe it makes sense to leave the "has 0 whatever". Do you consider it too verbose?

Reply:
I feel very verbose otherwise, but can be reverted

Reply:
Not sure at all, maybe we can leave it under the reason that people is used to see it.

Reply:
🤷

But for consistency, should notice everything else

Reply:
OK. Leave it like this :smile: